joplin/base/wagtail_hooks.py

```diff
@@ -193,5 +198,32 @@ def joplin_page_listing_more_buttons(page, page_perms, is_parent=False):
             _('Delete'),
             reverse('wagtailadmin_pages:delete', args=[page.id]),
             attrs={'title': _("Delete page '{title}'").format(title=page.get_admin_display_title())},
-            priority=30
         )
+
+
+
+@hooks.register('register_rich_text_features')
+def register_help_text_feature(features):
```

Comment:
I'm looking forward to see what other things we find regarding draftail.

I remember looking for draft-js table extensions before going to markdown route, it might be worth checking out some of those now that we have a clean implementation of a custom button working.

Reply:
We've got this issue in the backlog: 
https://github.com/cityofaustin/techstack/issues/2333

Luckily Thibaud wrote draftail and we know him now, so we could do some research and talk shop about a table inside draftail. 

Another route there might be some combo of using a Table Block: https://docs.wagtail.io/en/v2.6.1/reference/contrib/table_block.html
Inside of a react-streamfield so we could control the presentation more.